### PR TITLE
Fix some merge issues from the recent infra work

### PR DESCRIPTION
**Pull request by weshaggard**
cc @eerhardt 

core-setup official builds were broken because of some publish properties not getting updated. 

Also cleaned up some left over distrorid properties.

### CODE REVIEW

**Comment by weshaggard** [comment]
These should only impact the official builds so merging and queuing a build.

File: src/pkg/packaging/dir.proj [hunk 1]
```diff
@@ -141,10 +141,10 @@
     <Exec Command="tar -C $(SharedFrameworkPublishSymbolsDir) -czf $(AssetOutputPath)$(SharedFrameworkSymbolsCompressedFile) ."
           IgnoreExitCode="true"
           IgnoreStandardErrorWarningFormat="true" />
-    <Error Condition="!Exists('$(AssetOutputPath)$(CombinedCompressedFile)')" Message="Unable to create $(PackagesOutDir)$(CombinedCompressedFile)" />
-    <Error Condition="!Exists('$(AssetOutputPath)$(HostFxrCompressedFile)')" Message="Unable to create $(PackagesOutDir)$(HostFxrCompressedFile)" />
-    <Error Condition="!Exists('$(AssetOutputPath)$(SharedFrameworkCompressedFile)')" Message="Unable to create $(PackagesOutDir)$(SharedFrameworkCompressedFile)" />
-    <Error Condition="!Exists('$(AssetOutputPath)$(SharedFrameworkSymbolsCompressedFile)')" Message="Unable to create $(PackagesOutDir)$(SharedFrameworkSymbolsCompressedFile)" />
+    <Error Condition="!Exists('$(AssetOutputPath)$(CombinedCompressedFile)')" Message="Unable to create $(PackageOutputPath)$(CombinedCompressedFile)" />
```
Comment (eerhardt) [inline src/pkg/packaging/dir.proj]: These should be `$(AssetOutputPath)`

File: publish/dir.props [hunk 2]
```diff
@@ -21,24 +21,24 @@
       <ConditionalErrorMessage>Pushing took too long</ConditionalErrorMessage>
     </IgnorableErrorMessages>
   </ItemGroup>
-  
+
   <ItemGroup>
-    <CompressedFile Include="$(PackagesOutDir)**/*$(CompressedFileExtension)">
+    <CompressedFile Include="$(PackageOutputPath)**/*$(CompressedFileExtension)">
```
Comment (eerhardt) [inline publish/dir.props]: Should be `AssetOutputPath`.  In general - .nupkgs go to `PackageOutputPath`.  Everything else goes to `AssetOutputPath`.

File: src/pkg/packaging/dir.proj (hunk 1 above)
Comment (weshaggard) [reply src/pkg/packaging/dir.proj]: Good point I will update.

**Pull request merged**